api/models/workshop.py

```diff
@@ -23,6 +25,10 @@ class Workshop(CapacityMixin, Base):
         WorkshopDifficulty,
         verbose_name=_("Difficulty"),
     )
+    location = models.ForeignKey(
```

Comment:
@PetrDlouhy, it is unsure where the festival will be held every year. Do you remember Improtřesk on Klamovka? Improtřesk in Lužánky (Brno)? Do you remember using the Milevsko Nursery School building in 2015? Workshop locations have a great potential to every year. 2017 will definitely have different room dispositions to 2016.

Have it your way. Right now, your generating overhead. I'll change it when I have time.